app/src/main/java/com/example/androidapplicationtask/BindingAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.androidapplicationtask
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.squareup.picasso.Picasso
+
+
+@BindingAdapter("setBookmarkImage")
+fun ImageView.setBookmarkImage(flag: Int){
```

Comment:
change data type of flag from Int to Boolean

Reply:
done